base/applications/kbswitch/kbsdll/kbsdll.c

```diff
@@ -1,94 +1,86 @@
 /*
  * PROJECT:         ReactOS Keyboard Layout Switcher
  * FILE:            base/applications/kbswitch/kbsdll/kbsdll.c
- * PROGRAMMER:      Dmitry Chapyshev <dmitry@reactos.org>
- *
+ * PROGRAMMERS:     Dmitry Chapyshev <dmitry@reactos.org>
+ *                  Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
  */
 
 #include "../kbswitch.h"
 
-HHOOK hWinHook = NULL;
-HHOOK hShellHook = NULL;
-HINSTANCE hInstance = NULL;
-HWND hKbSwitchWnd = NULL;
+HINSTANCE g_hinstDLL = NULL;
+HHOOK g_hShellHook = NULL;
+HHOOK g_hCbtHook = NULL;
+HWND g_hwnd = NULL;
 
 static VOID
-PostMessageToMainWnd(UINT Msg, WPARAM wParam, LPARAM lParam)
+PostMessageToMainWnd(UINT uMsg, WPARAM wParam, LPARAM lParam)
```

Comment:
Could convert this to just a macro, or even, just call the PostMessage with the `g_hwnd` parameter directly where needed. I don't really see the need for having a separate function doing such a trivial action.

Reply:
Directly used `PostMessageW`.